frigidaire/__init__.py

```diff
@@ -202,13 +202,25 @@ def set_humidity(cls, humidity: int) -> List[Component]:
         return [Component(Setting.TARGET_HUMIDITY, humidity)]
 
     @classmethod
-    def set_temperature(cls, temperature: int) -> List[Component]:
```

Comment:
Rather than having two functions that do mostly the same thing, can we instead add another (optional) parameter to set_temperature to specify Unit and convert it to the proper representation here?

It would also simplify the code in home assistant since it'd go from an if/else to a simple mapping